Explorer/Assets/DCL/InWorldCamera/InWorldCamera/UI/InWorldCameraController.cs

```diff
@@ -0,0 +1,155 @@
+﻿using Arch.Core;
+using Cysharp.Threading.Tasks;
+using DCL.Audio;
+using DCL.CharacterCamera;
+using DCL.ExplorePanel;
+using DCL.InWorldCamera.CameraReelStorageService;
+using DCL.InWorldCamera.CameraReelStorageService.Schemas;
+using DCL.InWorldCamera.InWorldCamera.Playground;
+using DCL.UI;
+using ECS.Abstract;
+using JetBrains.Annotations;
+using MVC;
+using System.Diagnostics;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.InWorldCamera.ScreencaptureCamera.UI
+{
+    public class InWorldCameraController : ControllerBase<InWorldCameraView>
```

Comment:
Please provide some documentation about what's this UI about, what's its main purpose or when is it displayed.